.travis.yml

```diff
@@ -22,10 +27,8 @@ install:
   - pip install -e .[qa,rest]
   - pip install -U $DJANGO
 
-before_script:
-  - ./run-qa-checks
-
 script:
+  - ./run-qa-checks
```

Comment:
why this change? Is it necessary? If not, please rollback, we should not change code that works without a good reason.

Reply:
I too think that this might not be necessary. But after going through travis build cycle, I came to know that after_failure tests only for the error in the scripts. If build fails due to any other code logic, then we might not be able to get the reason for error.

Too verify the same, I tests this on my own repository and here are the results.
[
![travis_error](https://user-images.githubusercontent.com/42766576/91872774-e1c09b80-ec95-11ea-8b4c-de4d1e0b6df4.png)
](url)

Reply:
ok, great, thanks for the explanation, now makes sense, but please add a brief comment before ./run-qa-checks, eg:

```
# check here to allow travisbuddy to report what went wrong in case of failure
./run-qa-checks
```

Reply:
@nemesisdesign done. I added some comments to give a brief.